plugins/toolkit/jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -195,6 +195,7 @@
         <statusBarWidgetFactory id="AwsSettingsPanel" implementation="software.aws.toolkits.jetbrains.core.credentials.AwsSettingsPanelInstaller"/>
         <statusBarWidgetFactory id="CawsSpaceProjectInfo" implementation="software.aws.toolkits.jetbrains.services.caws.CawsStatusBarInstaller"/>
 
+        <postStartupActivity implementation="software.aws.toolkits.jetbrains.services.telemetry.AwsToolkitStartupMetrics" order="first"/>
```

Comment:
I wonder if we should skip the order here? Would it interfere with any other JB specific activities?